core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java

```diff
@@ -249,15 +245,15 @@ public Void call(BlockWorkerClientService.Client client)
   }
 
   @Override
-  public String requestBlockLocation(final long blockId, final long initialBytes)
-      throws IOException {
+  public String requestBlockLocation(final long blockId, final long initialBytes,
+      final int writeTier) throws IOException {
```

Comment:
Oh, awesome! Do you guys mind if work extending this feature for remote writes using the new protocol?